style.css

```diff
@@ -0,0 +1,99 @@
+@import url('https://fonts.cdnfonts.com/css/Roboto');
+@import url('https://fonts.cdnfonts.com/css/poppins');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+.top_header {
+  background: #3c3a39;
+  width: 375px;
+}
```

Comment:
- [ ] your design for the tool bar section containing the logo and nav seems to be inconsistent with figma design as shown below with regards to the height.
> figma design
![header-portfolio](https://user-images.githubusercontent.com/89055502/234738459-1b8c4f5a-e1eb-4992-8c65-7fdc01ce792a.png)

> student design
![student-2](https://user-images.githubusercontent.com/89055502/234738701-82fbb087-48f8-4d22-bb93-5c3f026c75d7.png)

- [ ] from the figma template there is a fixed height as shown below which you are not implementing, kindly please add this to your code base to improve the overall structure. :+1: 
![toolbar-height](https://user-images.githubusercontent.com/89055502/234738882-5a3b29d0-8bbd-463d-91a3-90afc073a400.png)

Reply:
@Efiamotu-1 , height implemented for both , thanks